packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -71,100 +69,98 @@ export const Stackbox = () => {
   }, [frequency]);
 
   return (
-    <>
-      <div className="max-w-lg mx-auto my-24 bg-white shadow-2xl rounded-2xl">
-        <div className="px-5 py-4 border shadow-lg border-surface-50 rounded-2xl">
-          <div className="flex items-end justify-between pb-4 border-b border-surface-50">
-            <SelectTokenButton
-              label="Deposit from"
-              onClick={openTokenPicker}
-              token={tokenFrom}
-            />
-            <Icon
-              name="arrow-left"
-              className="flex items-center justify-center p-2 w-16 h-9 bg-surface-50 rounded-2xl rotate-180"
-            />
-            <SelectTokenButton
-              label="To receive"
-              onClick={openTokenPicker}
-              token={tokenTo}
-            />
-          </div>
-          <div className="py-2">
-            <input
-              type="number"
-              placeholder="0.0"
-              className="w-full py-3 text-4xl font-semibold outline-none"
-            />
-          </div>
+    <div className="max-w-lg mx-auto my-24 bg-white shadow-2xl rounded-2xl">
```

Comment:
I don't think this style belongs to the component. This style is related to the page. If we want to have a Stackbox inside a Modal, for example, we can't have this style. This mental exercise helps me defining where each style belongs (inside the component or inside the page). But what do you think? :)

Talking about `max-w-lg mx-auto my-24`.

Reply:
OIC I thought the Stackbox WAS kinda the page content itself. I'll refactor this one, thanks for the heads up!